src/lambda.ts

```diff
@@ -117,7 +117,7 @@ export const serve: ServeHandler = (nameOrInngest, fns, opts) => {
         body,
         statusCode: status,
         headers,
-      });
+      } as APIGatewayProxyResult);
```

Comment:
This is hiding the fact that `body` can be a `ReadableStream`. 🤮

I don't like that these response transformers now have to _maybe_ deal with streaming. Perhaps a better solution would be to provide a second response transformer if streaming is supported, in which case we can also easily ascertain internally whether a handler supports it or not.

This also likely means we can safely make edge streaming opt-out instead of opt-in, as handlers themselves can declare support rather than having to assume it'll work.